hub/saml-soap-proxy/src/test/java/uk/gov/ida/hub/samlsoapproxy/healthcheck/HealthCheckDataTest.java

```diff
@@ -3,52 +3,50 @@
 import org.junit.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 
 public class HealthCheckDataTest {
 
     @Test
     public void shouldReturnEmptyHealthCheckDataWhenExtractedFromNullString() {
         HealthCheckData testData = HealthCheckData.extractFrom("test-id");
-        assertFalse(testData.getVersion().isPresent());
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
+        assertThat(testData.getVersion().isPresent()).isFalse();
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
     }
 
     @Test
     public void shouldReturnEmptyHealthCheckDataWhenExtractedFromEmptyString() {
         HealthCheckData testData = HealthCheckData.extractFrom("");
-        assertFalse(testData.getVersion().isPresent());
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
+        assertThat(testData.getVersion().isPresent()).isFalse();
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
     }
 
     @Test
     public void shouldReturnEmptyHealthCheckDataWhenExtractedFromSimpleIdString() {
         HealthCheckData testData = HealthCheckData.extractFrom("test-id");
         assertThat(testData.getVersion());
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
     }
 
     @Test
     public void shouldReturnHealthCheckDataWithVersionWhenExtractedFromStringThatContainsVersionData() {
         String expectedVersion = "1234";
         HealthCheckData testData = HealthCheckData.extractFrom("test-id-version-" + expectedVersion);
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
-        assertTrue(testData.getVersion().get().contains(expectedVersion));
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
+        assertThat(testData.getVersion().get().contains(expectedVersion));
```

Comment:
Can we replace `assertThat(testData.getVersion().get().contains(expectedVersion));` with `assertThat(testData.getVersion().get().contains(expectedVersion)).isTrue();`?

Reply:
It doesn't make much of a difference  - the fluent assertion message wouldn't contain any more detail so I think it's not necessary.

Reply:
It does make difference. I tried the original code to make it fail by making the version different. It didn't fail.

I added `.isTrue()` and ran the following code again. It failed. This showed that the test was working because `.isTrue()` was added.

```
    @Test
    public void shouldReturnHealthCheckDataWithVersionWhenExtractedFromStringThatContainsVersionData() {
        String expectedVersion = "1234";
        HealthCheckData testData = HealthCheckData.extractFrom("test-id-version-" + "1235");
        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
        assertThat(testData.getVersion().get().contains(expectedVersion)).isTrue();
    }
```

Reply:
We should use `assertThat(testData.getVersion().get()).contains(expectedVersion);` instead of `assertThat(testData.getVersion().get().contains(expectedVersion));` It gives better failure message.